pkgs/top-level/ocaml-packages.nix

```diff
@@ -297,10 +299,14 @@ let
 
     ocamlfuse = callPackage ../development/ocaml-modules/ocamlfuse { };
 
+    ocaml_gettext = callPackage ../development/ocaml-modules/ocaml-gettext { };
```

Comment:
No. ```gettext``` is taken for C library which ```ocaml_gettext``` depends on. Using ```gettext``` here results in "infinite recursion" error. The same for ```libvirt```.